backend/de.metas.business/src/main/java/de/metas/order/impl/AbstractOrderDAO.java

```diff
@@ -524,4 +525,17 @@ public Stream<I_C_Order> streamOrders(@NonNull final IQueryFilter<I_C_Order> ord
 				.create()
 				.iterateAndStream();
 	}
+
+	@Override
+	public Stream<I_C_OrderLine> streamOrderLines(@NonNull final OrderLineQuery query)
```

Comment:
I would use something like "streamByModularPurchaseContractId" and pass only the flatrate term ID instead of the query.

Reply:
I did this, so it is extendable as it returns a common usable I_C_OrderLine stream